<launch>
  <node pkg="spur_controller" name="spur_base_controller"
        type="base_controller.py" output="screen" ns="/spur">
    <remap from="cmd_vel" to="/cmd_vel" />
    <remap from="odom" to="/odom" />
  </node>
  <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
    <param name="output_frame" value="odom"/>
    <param name="freq" value="30.0"/>
    <param name="sensor_timeout" value="1.0"/>
    <param name="odom_used" value="true"/>
    <param name="imu_used" value="false"/>
    <param name="vo_used" value="false"/>
    <param name="debug" value="true"/>
    <param name="self_diagnose" value="false"/>
  </node>
</launch>
